src/evergreen/api.py

```diff
@@ -222,6 +223,16 @@ def patches_by_project(self, project_id, params=None):
         patches = self._lazy_paginate_by_date(url, params)
         return (Patch(patch, self) for patch in patches)
 
+    def commit_queue_for_project(self, project_id):
+        """
+        Get the current commit queue for the specified project.
+
+        :param project_id: Id of project to query.
+        :return: Current commit queue for project.
+        """
+        url = self._create_url('/projects/{project_id}/commit_queue'.format(project_id=project_id))
+        return CommitQueue(self._paginate(url), self)
```

Comment:
Question for my own understanding -- does this fetch a CommitQueue object from the database or instantiate a new instance of one?

Reply:
The `self._paginate` will return a json object from the evergreen api. We then use that json to create a `CommitQueue` object that we are returning here.